examples/headsail-bsp/sdram_hpc.x

```diff
@@ -1,12 +1,12 @@
 MEMORY
 {
   BOOTRAM : ORIGIN = 0x10000, LENGTH = 0x8000
-  SDRAM : ORIGIN = 0x100000000, LENGTH = 0x70000000
+  SDRAM : ORIGIN = 0x104080000, LENGTH = 0x1bf80000
```

Comment:
We think the SDRAM is actually available starting at 0x1_0000_0000. The memory map just has some incorrect tags that overlap it. Those tags need to be corrected or disabled.

Reply:
Memory map adjusted as discussed + HPC bootram external addressing from HPC fixed in headsail.repl